test/smoke_test/smoke_test.py

```diff
@@ -38,14 +24,13 @@ def main(args=None) -> None:
         logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.DEBUG)
 
     _chdir()
-    _run_smoke_test(options.ffmpeg)
+    _run_smoke_test()
 
 
 def _parse_args(args):
     parser = argparse.ArgumentParser()
 
     # Warning: Please note this option should not be widely used, only use it when absolutely necessary
-    parser.add_argument("--no-ffmpeg", dest="ffmpeg", action="store_false")
```

Comment:
let's keep this line because I think some of the scripts (from other repos) are potentially relying on it still. But we should edit the comment above and say "this does nothing, only exists for BC"